java/build.sh

```diff
@@ -0,0 +1,7 @@
+export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
```

Comment:
Can we put this logic in the top-level build.sh please? We do that with all of the other language support.